pkgs/development/libraries/qt-6/modules/qtbase/default.nix

```diff
@@ -278,7 +279,9 @@ stdenv.mkDerivation rec {
     ]
     ++ lib.optional (
       qttranslations != null && !isCrossBuild
-    ) "-DINSTALL_TRANSLATIONSDIR=${qttranslations}/translations";
+    ) "-DINSTALL_TRANSLATIONSDIR=${qttranslations}/translations"
+    # https://bugreports.qt.io/browse/QTBUG-136060
+    ++ lib.optional stdenv.hostPlatform.isLinux "-DCMAKE_SYSTEM_VERSION=${linuxKernel.kernels.linux_default.version}";
```

Comment:
Can we just set this to "nixpkgs" or something? Arbitrary kernel version is pretty silly. Also, does it have to be Linux specific?

Reply:
I'm not opposed to that, I was just trying to be conservative ;).

If we're confident this is not functionally used anywhere (which I agree does seem like a relatively safe assumption) I'd be happy to fixate it to `nixpkgs`...

Reply:
Actually looks like the right way to avoid this is just empty string: https://github.com/qt/qtbase/blob/616d33c87ab6dd77809ee56e39e08eed30089130/cmake/QtModuleHelpers.cmake#L1528

Reply:
right, since it's actually "supported" there that seems like a reasonable thing to do.